app/src/main/java/com/practicum/playlistmarket/player/ui/activity/MediaPlayerActivity.kt

```diff
@@ -29,10 +29,9 @@ class MediaPlayerActivity : AppCompatActivity() {
 
     private lateinit var binding: ActivityMediaPlayerBinding
 
-    private lateinit var viewModel: MediaPlayerViewModel
-
     private var songUrl: String = ""
 
+    private val vm : MediaPlayerViewModel by viewModel()
```

Comment:
```suggestion
    private val viewModel : MediaPlayerViewModel by viewModel()
```
Лучше сохранить принятый нейминг вьюмоделей